docs/reference/query-dsl/query-string-query.asciidoc

```diff
@@ -107,11 +110,6 @@ the query string. This allows to use a field that has a different analysis chain
 for exact matching. Look <<mixing-exact-search-with-stemming,here>> for a
 comprehensive example.
 
-|`all_fields` | Perform the query on all fields detected in the mapping that can
```

Comment:
Should we leave this in here, but mark it as deprecated? It still technically exists and I'm not sure we want people that have `all_fields: true` set to not be able to look up what the option does (until it's removed)